packages/perspective-viewer-d3fc/src/js/charts/ohlcCandle.js

```diff
@@ -37,19 +37,20 @@ function ohlcCandle(seriesSvg) {
             .settings(settings)
             .scale(keys.length > 1 ? upColor : null);
 
-        const series = seriesSvg()
+        const series = seriesCanvas()
             .crossValue(d => d.crossValue)
             .openValue(d => d.openValue)
             .highValue(d => d.highValue)
             .lowValue(d => d.lowValue)
             .closeValue(d => d.closeValue)
-            .decorate(selection => {
-                selection.style("fill", d => (isUp(d) ? upColor(d.key) : downColor(d.key)));
-                selection.style("stroke", d => (isUp(d) ? upColor(d.key) : downColor(d.key)));
+            .decorate((context, d) => {
+                const color = isUp(d) ? upColor(d.key) : downColor(d.key);
```

Comment:
Nice

Reply:
If we don't switch to canvas, maybe we should make this refactor anyway?

Reply:
Definitely